src/iDealAdvancedConnector/iDealConnectorOptions.cs

```diff
@@ -19,6 +19,11 @@ public class IDealConnectorOptions
         /// </summary>
         /// <value></value>
         public string AcquirerCertificate { get; set; }
+
+        /// <summary>
+        /// Acquirer timeout in seconds
+        /// </summary>
+        /// <value></value>
         public string AcquirerTimeout { get; set; }
```

Comment:
should we remove `AcquirerTimeout` altogether?

Reply:
Ok, it was just it was no longer used within this project itself

Reply:
yeah true it's a bit awkward. would documentation suffice for the moment?